airbyte-integrations/connectors/destination-kafka/src/main/resources/spec.json

```diff
@@ -0,0 +1,221 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/destinations/kafka",
+  "supportsIncremental": true,
```

Comment:
We should add the two new attributes here. If you pull in the latest master, you would see it in other destination spec files.
```
  "supportsNormalization": false,
  "supportsDBT": false,
  ```

Reply:
Done!